package.yaml

```diff
@@ -133,7 +133,6 @@ executables:
       - -Werror=missing-fields
       - -Wredundant-constraints
       - -O
-      - -threaded
```

Comment:
Please only do this for the broken platform (Windows in this case).

And please add a comment why it is needed so somebody does not introduce it later.